docs(editorial policy)

alyonurchick1 opened the pull request:
closes consta-design-system/uikit#3760

Добавленные разделы:
- [Форматирование текста](https://refs-pull-52-merge---portal.consta.design/libs/portal/portal-formatting-rules)
- [Виды разделов](https://refs-pull-52-merge---portal.consta.design/libs/portal/portal-types-of-sections)
- [Структура документации](https://refs-pull-52-merge---portal.consta.design/libs/portal/portal-documentation-structure)
- [Работа с документацией](https://refs-pull-52-merge---portal.consta.design/libs/portal/portal-settin-up-documentation)
- [Работа с Git](https://refs-pull-52-merge---portal.consta.design/libs/portal/portal-work-with-git)
- [Глоссарий](https://refs-pull-52-merge---portal.consta.design/libs/portal/portal-glossary)
- [Соглашение об именах](https://refs-pull-52-merge---portal.consta.design/libs/portal/portal-naming-rules)

Разделы будут обновляться по мере добавления новых разделов в связи с:
1. Обеспечением корректного перехода по ссылкам между разделами
2. Внедрением нового компонента `Informer` в доку
3. Обеспечением консистентности
4. Вычиткой и редактированием

!Не заливать на портал до момента добавления всех разделов. Я отпишусь, когда добавлю всё

alyonurchick1 committed:
docs(editorial policy): add formatting rules

Closes #3760

alyonurchick1 commented:
@gizeasy хочу заменить пример компонента на какой-нибудь более простой и короткий. посмотри, пожалуйста, как выйдешь с больничного

alyonurchick1 committed:
docs(editorial policy): add types of section

alyonurchick1 committed:
Update typesOfSections.stand.mdx

alyonurchick1 committed:
Update typesOfSections.stand.mdx

alyonurchick1 commented:
@linturaven , посмотри, пожалуйста, получится ли переделать эту схему так, чтобы она красиво и не громоздко отображалась на портале. А то сейчас из-за того, что вертикально, всё место занимает на странице.
![image](https://github.com/user-attachments/assets/a107021f-3ee9-4151-b5aa-8f71cc3e4e8b)

alyonurchick1 committed:
docs(editorial policy): add documentation structure

alyonurchick1 committed:
docs(editorial policy): add setting up documentation

edit documentation structure and add setting up documentation

alyonurchick1 committed:
Update settingUpDocumentation.stand.mdx

gizeasy commented on src/docs/documentationStructure/documentationStructure.stand.mdx:
```diff
@@ -0,0 +1,181 @@
+
+## Настройки раздела
+
+В файле с раширением `.tsx` вы можете задать настройки раздела:
```
раширением -> расширением

gizeasy commented on src/docs/documentationStructure/documentationStructure.stand.mdx:
```diff
@@ -0,0 +1,181 @@
+Документация состоит из двух типов файлов:
+
+- [страница раздела](#страницы-раздела) – файл с расширением `.mdx`.
+- [настройки раздела](#настройки-раздела) – файл с расширением `.tsx`.
```
Давай будем писать про расширение `.ts`. `.tsx` - используется в как обозначение `jsx` синтаксиса а в настройках его обычно нет. Просто сборка по может кушать и `.ts`, и`.tsx`. Но правельнее использовать `.ts`

gizeasy commented on src/docs/documentationStructure/documentationStructure.stand.mdx:
```diff
@@ -0,0 +1,181 @@
+
+Где:
+
+- `title` – название страницы. Оно будет отображаться вверху страницы в качестве заголовка H1.
```
Заголовок страницы

gizeasy commented on src/docs/documentationStructure/documentationStructure.stand.mdx:
```diff
@@ -0,0 +1,181 @@
+- `group` – группа (раздел), к которой принадлежит страница. Все группы разделов можно посмотреть в папке `stand` в файле `standConfig.ts`.
+- `order` – порядок отображения страницы на портале. Так, чтобы отобразить страницу в первом разделе второй, задайте значение `120`.
+- `alias` – ключевые слова для поиска по порталу.
+- `description` – краткое описание содержимого страницы. Отображается на портале внизу названия страницы.
```
Отображается на портале внизу названия страницы. 
->
Отображается под заголовком страницы.

gizeasy commented on src/docs/documentationStructure/documentationStructure.stand.mdx:
```diff
@@ -0,0 +1,181 @@
+
+Где:
+
+- `title` – название страницы. Оно будет отображаться вверху страницы в качестве заголовка H1.
```
Заголовок

gizeasy commented on src/docs/documentationStructure/documentationStructure.stand.mdx:
````diff
@@ -0,0 +1,181 @@
+<>
+
+```jsx
+import { createStand } from '##/stand/standConfig';
````
Кажется что не нужно спутывать понятия раздел, страница, группа.

createStand - создает страницу, страница может быть о компоненте а может в общем рассказывать о принципах(и т.п.)

Страница существует в группе, группа в библиотеке.

gizeasy commented on src/docs/documentationStructure/documentationStructure.stand.mdx:
```diff
@@ -0,0 +1,181 @@
+</>
+</MdxTabs>
+
+## Страницы раздела
```
Вкладки страницы

gizeasy commented on src/docs/documentationStructure/images/description.png:
удали

gizeasy commented on src/docs/documentationStructure/images/section.png:
удали

gizeasy commented on src/docs/documentationStructure/documentationStructure.stand.mdx:
```diff
@@ -0,0 +1,181 @@
+- [страница раздела](#страницы-раздела) – файл с расширением `.mdx`.
+- [настройки раздела](#настройки-раздела) – файл с расширением `.tsx`.
+
+## Настройки раздела
```
ввести понятие стенда
раздел это стенд

 .stand.mdx - главная страница стенда
 dev.stand.mdx и прочие tab.stand.mdx - второстипеные страницы

alyonurchick1 committed:
Создала отдельную папку для раздела Portal

alyonurchick1 committed:
Update settingUpDocumentation.stand.mdx

alyonurchick1 committed:
Работа с гитом

alyonurchick1 committed:
Update documentationStructure.stand.mdx

alyonurchick1 committed:
glossary

alyonurchick1 committed:
Подняла версию

alyonurchick1 committed:
создала папки

alyonurchick1 committed:
подняла версию

alyonurchick1 committed:
Update formattingRules.stand.mdx

Описала компонент информер в разделе с форматированием. Пытылась использовать его в добавленных разделов, но из-за отсутствия возможности форматировать текст внутри информера отложила эту идею до следующего обновления.
@gizeasy оч важно дать возможность форматировать текст. Даже ссылки не вставляются

alyonurchick1 committed:
add notes

alyonurchick1 committed:
Update glossary.stand.mdx

alyonurchick1 committed:
Update package.json

alyonurchick1 committed:
Merge branch 'master' into docs/editorial-policy

alyonurchick1 committed:
Update package.json

alyonurchick1 committed:
Update glossary.stand.mdx

alyonurchick1 committed:
add professional jargon

alyonurchick1 committed:
Update glossary.stand.mdx

alyonurchick1 committed:
up version

alyonurchick1 committed:
update

alyonurchick1 committed:
add naming rules

alyonurchick1 committed:
Update namingRules.stand.mdx

alyonurchick1 committed:
Update namingRules.stand.mdx

alyonurchick1 committed:
update

alyonurchick1 committed:
edit setting up

gizeasy committed:
Update package.json

alyonurchick1 committed:
update

alyonurchick1 committed:
update

alyonurchick1 commented:
мб не "предупреждение", а "важно"

alyonurchick1 commented:
https://refs-pull-52-merge---portal.consta.design/libs/portal/portal-formatting-rules?hash=%D0%BF%D1%80%D0%B8%D0%BC%D0%B5%D1%87%D0%B0%D0%BD%D0%B8%D0%B5  -сломалась верстка  в примечаниях

alyonurchick1 commented:
убрать блок "Заметки"

alyonurchick1 committed:
Merge branch 'master' into docs/editorial-policy

alyonurchick1 committed:
update

alyonurchick1 committed:
union pages

alyonurchick1 committed:
union

gizeasy committed:
Update package.json

gizeasy committed:
Update package.json

alyonurchick1 committed:
Merge branch 'master' into docs/editorial-policy

alyonurchick1 committed:
add props naming rules

gizeasy committed:
Update deploy.yml

alyonurchick1 committed:
Update namingRules.stand.mdx

alyonurchick1 committed:
Merge branch 'docs/editorial-policy' of https://github.com/consta-design-system/portal into docs/editorial-policy

alyonurchick1 committed:
Update namingRules.stand.mdx

gizeasy committed:
Merge branch 'master' into docs/editorial-policy
